docs/docs/reference/cloud/monorepos.md

```diff
@@ -0,0 +1,34 @@
+---
+title: "Monorepos"
+description: "Monorepos support in Gatsby Cloud"
+---
+
+## Introduction
+
+Gatsby Cloud has support for monorepos projects, but since there is a plethora of dependency management tools out there some issues can occure.
+This document will walk you through the tools that are supported and how to troubleshoot some edge cases
+
+## Monorepo Support
+
+The following tables shows the level of support for each tool
+
+**Legend**
+
+| Icon | Feature Capability                       |
+| ---- | ---------------------------------------- |
+| ●    | Fully Supported                          |
+| ◐    | Somewhat Supported (support is minimal ) |
+| ○    | Not Supported                            |
+
+**Support**
+
+| Tool.                            | Level of Support | Notes                                                   |
+| -------------------------------- | ---------------- | ------------------------------------------------------- |
+| Yarn Workspaces (v1)             | ◐                |                                                         |
+| Yarn Workspaces (v2/v3 with PnP) | ◐                |                                                         |
+| Lerna                            | ●                |                                                         |
+| NPM Workspaces (v7 and onwards)  | ●                |                                                         |
+| NX                               | ○                | Work is in progress                                     |
+| Turborepo                        | ○                | We do not have plans to support Turborepo at this time. |
+
+## Troubleshooting
```

Comment:
Completely missing?

Reply:
The only thing I can put here for now is redirecting people to support when they have issues and then we can actively add more stuff to it when we discover any edge case

Reply:
Can we add something around pnp? If you're having issues use `nodeLinker: node-modules`

https://yarnpkg.com/features/pnp/#pnp-loose-mode
https://yarnpkg.com/getting-started/migration/#if-required-enable-the-node-modules-plugin
https://yarnpkg.com/getting-started/recipes/#hybrid-pnp--node_modules-mono-repo

Reply:
Can we add something around pnp? If you're having issues use `nodeLinker: node-modules`

https://yarnpkg.com/features/pnp/#pnp-loose-mode
https://yarnpkg.com/getting-started/migration/#if-required-enable-the-node-modules-plugin
https://yarnpkg.com/getting-started/recipes/#hybrid-pnp--node_modules-mono-repo